config/environments/test.rb

```diff
@@ -42,17 +42,4 @@
 
   # Print deprecation notices to the stderr.
   config.active_support.deprecation = :stderr
-
-  # Raises error for missing translations
-  # config.action_view.raise_on_missing_translations = true
```

Comment:
maybe I'll find out later in the PR, but this config `raise_on_missing_translations` isn't required any more?

Reply:
Ahh, that got removed by accident when I removed the AMQP configuration, I saw that it was a comment, and because there was no whitespace between it and the configuration below, I must have assumed it was about the amqp configuration.

It has no effect, because its commented out, but I'll restore it anyway as it tends to make Rails upgrades easier if you've kept modification of the environments/*.rb files to a minimum.

Reply:
Restored